src/modules/client/components/customers/CustomerProfileHeader.vue

```diff
@@ -35,31 +47,56 @@ export default {
   },
   components: {
     'ni-button': Button,
+    'stop-support-modal': StopSupportModal,
   },
   data () {
     return {
       STATUS_TYPES,
       get,
+      getStartOfDay,
+      modalLoading: false,
+      stopSupportModal: false,
+      newStatus: { stopReason: '', stoppedAt: '' },
     };
   },
   computed: {
     ...mapState('customer', ['customer']),
+    status () {
+      if (this.customer.archivedAt) return ARCHIVED;
+      if (this.customer.stoppedAt) return STOPPED;
+
+      return ACTIVATED;
+    },
     statusDate () {
-      switch (get(this.customer, 'status.value')) {
-        case ACTIVATED: return formatDate(this.customer.status.activatedAt);
-        case STOPPED: return formatDate(this.customer.status.stoppedAt);
-        case ARCHIVED: return formatDate(this.customer.status.archivedAt);
+      switch (this.status) {
+        case ACTIVATED: return formatDate(this.customer.createdAt);
+        case STOPPED: return formatDate(this.customer.stoppedAt);
+        case ARCHIVED: return formatDate(this.customer.archivedAt);
         default: return 'N/A';
       }
     },
     relativeStatusDate () {
-      switch (get(this.customer, 'status.value')) {
-        case ACTIVATED: return formatDateDiff(dateDiff(this.customer.status.activatedAt, new Date()));
-        case STOPPED: return formatDateDiff(dateDiff(this.customer.status.stoppedAt, new Date()));
-        case ARCHIVED: return formatDateDiff(dateDiff(this.customer.status.archivedAt, new Date()));
+      switch (this.status) {
+        case ACTIVATED: return formatDateDiff(dateDiff(this.customer.createdAt, new Date()));
+        case STOPPED: return formatDateDiff(dateDiff(this.customer.stoppedAt, new Date()));
+        case ARCHIVED: return formatDateDiff(dateDiff(this.customer.archivedAt, new Date()));
         default: return '';
       }
     },
+    minStoppingDate () {
+      return getStartOfDay(new Date(this.customer.createdAt)).toISOString();
```

Comment:
je pense que tu peux simplifier en 
`return getStartOfDay(this.customer.createdAt).toISOString();`

Reply:
ça marche chez toi ? parce que chez moi vu que tu createdAt est une string, si j'essaie de lui appliquer getstartofday qui attend une date ça plante